FPU Warning on ARM machines

## Issue Details

When running tests on our arm mac machine, I get those warnings :
```
/Users/magritte/cgal/Number_types/include/CGAL/FPU.h:480:21: warning: value size does not match register size specified by the constraint and modifier [-Wasm-operand-widths]
    CGAL_IA_GETFPCW(cw);
                    ^
/Users/magritte/cgal/Number_types/include/CGAL/FPU.h:480:5: note: use constraint modifier "w"
    CGAL_IA_GETFPCW(cw);
    ^
/Users/magritte/cgal/Number_types/include/CGAL/FPU.h:452:48: note: expanded from macro 'CGAL_IA_GETFPCW'
#define CGAL_IA_GETFPCW(CW) asm volatile ("MRS %0, FPCR" : "=r" (CW))
                                               ^
/Users/magritte/cgal/Number_types/include/CGAL/FPU.h:490:19: warning: value size does not match register size specified by the constraint and modifier [-Wasm-operand-widths]
  CGAL_IA_SETFPCW(cw);
                  ^
/Users/magritte/cgal/Number_types/include/CGAL/FPU.h:490:3: note: use constraint modifier "w"
  CGAL_IA_SETFPCW(cw);
  ^
/Users/magritte/cgal/Number_types/include/CGAL/FPU.h:451:54: note: expanded from macro 'CGAL_IA_SETFPCW'
#define CGAL_IA_SETFPCW(CW) asm volatile ("MSR FPCR, %0" : :"r" (CW))
                                                     ^
```
This appears when CMAKE_NO_SYSTEM_FROM_IMPORTED is ON. 

@mglisse, do you have any idea what is wrong here ?
## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Mac
* Compiler:Apple clang version 12.0.0 (clang-1200.0.32.29)
* Release or debug mode:Debug
* Specific flags used (if any):CMAKE_NO_SYSTEM_FROM_IMPORTED=TRUE
* CGAL version:all

--- a/Number_types/include/CGAL/FPU.h
+++ b/Number_types/include/CGAL/FPU.h
@@ -450,7 +450,7 @@ typedef unsigned int FPU_CW_t;
 # elif defined  __aarch64__
 #define CGAL_IA_SETFPCW(CW) asm volatile ("MSR FPCR, %0" : :"r" (CW))
 #define CGAL_IA_GETFPCW(CW) asm volatile ("MRS %0, FPCR" : "=r" (CW))
-typedef unsigned int FPU_CW_t;
+typedef unsigned long FPU_CW_t;
 #define CGAL_FE_TONEAREST    (0x0)
 #define CGAL_FE_TOWARDZERO   (0xC00000)
 #define CGAL_FE_UPWARD       (0x400000)

